test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -2251,24 +2251,36 @@ class ClientLbInterceptTrailingMetadataTest : public ClientLbEnd2endTest {
     return std::move(load_report_);
   }
 
+  void WaitForLbCallback() {
+    grpc::internal::MutexLock lock(&mu_);
+    while (!trailer_intercepted_) {
+      cond_.WaitWithTimeout(&mu_, absl::Seconds(3));
```

Comment:
I think the `WaitForLbCallback()` method should return a bool, which should be false if `WaitWithTimeout()` returns false.  That way, the caller can say `ASSERT_TRUE(WaitForLbCallback())`, and the test will fail if no callback arrives on time.

Reply:
Oh I didn't notice WaitWithTimeout() returned a bool - done, thanks!